server/src/main/scala/slickdb/Tables.scala

```diff
@@ -121,60 +40,119 @@ trait Tables {
     val scheduled_date: Rep[java.sql.Timestamp] = column[java.sql.Timestamp]("scheduled_date")
-    /** Database column gate SqlType(text), Default(None) */
     val day_of_week: Rep[Int] = column[Int]("day_of_week")
-    /** Database column gate SqlType(text), Default(None) */
     val week_of_year: Rep[Int] = column[Int]("week_of_year")
-    /** Database column stand SqlType(text), Default(None) */
     val document_type: Rep[String] = column[String]("document_type")
-    /** Database column status SqlType(text) */
     val document_issuing_country_code: Rep[String] = column[String]("document_issuing_country_code")
-    /** Database column scheduled SqlType(timestamp) */
     val eea_flag: Rep[String] = column[String]("eea_flag")
-    /** Database column estimated SqlType(timestamp), Default(None) */
     val age: Rep[Int] = column[Int]("age")
-    /** Database column actual SqlType(timestamp), Default(None) */
     val disembarkation_port_code: Rep[String] = column[String]("disembarkation_port_code")
-    /** Database column estimatedchox SqlType(timestamp), Default(None) */
     val in_transit_flag: Rep[String] = column[String]("in_transit_flag")
-    /** Database column actualchox SqlType(timestamp), Default(None) */
     val disembarkation_port_country_code: Rep[String] = column[String]("disembarkation_port_country_code")
-    /** Database column pcp SqlType(timestamp) */
     val nationality_country_code: Rep[String] = column[String]("nationality_country_code")
-    /** Database column totalpassengers SqlType(int4), Default(None) */
     val passenger_identifier: Rep[String] = column[String]("passenger_identifier")
-    /** Database column pcppassengers SqlType(int4), Default(None) */
     val in_transit: Rep[Boolean] = column[Boolean]("in_transit")
+    val json_file: Rep[String] = column[String]("json_file")
   }
+
+  /** Table description of table arrival. Objects of this class serve as prototypes for rows in queries. */
+  class Arrival(_tableTag: Tag) extends {
+    private val maybeSchema = profile match {
+      case _: PostgresProfile => Some("public")
+      case _ => None
+    }
+  } with profile.api.Table[ArrivalRow](_tableTag, maybeSchema, "arrival") {
+    def * = (code, number, destination, origin, terminal, gate, stand, status, scheduled, estimated, actual, estimatedchox, actualchox, pcp, totalpassengers, pcppassengers,scheduled_departure) <> (ArrivalRow.tupled, ArrivalRow.unapply)
+    def ? = (Rep.Some(code), Rep.Some(number), Rep.Some(destination), Rep.Some(origin), Rep.Some(terminal), gate, stand, Rep.Some(status), Rep.Some(scheduled), estimated, actual, estimatedchox, actualchox, Rep.Some(pcp), totalpassengers, pcppassengers,scheduled_departure).shaped.<>({ r=>import r._; _1.map(_=> ArrivalRow.tupled((_1.get, _2.get, _3.get, _4.get, _5.get, _6, _7, _8.get, _9.get, _10, _11, _12, _13, _14.get, _15, _16,_17)))}, (_:Any) =>  throw new Exception("Inserting into ? projection not supported."))
```

Comment:
too long for single line

Reply:
These long lines are auto-generated code from slick. I've deleted the longer one as it's not used, but the first one is required